addons/website/views/website_navbar_templates.xml

```diff
@@ -3,7 +3,7 @@
     <!-- publish/unpublish button -->
     <template id="website_publisher" name="Website Publisher">
         <t groups="website.group_website_publisher" t-if="'website_published' in main_object.fields_get() and ('can_publish' not in main_object.fields_get() or main_object.can_publish)">
-            <div t-attf-class="js_publish_management #{extra_classes} #{main_object.website_published and 'css_published' or 'css_unpublished'}" t-att-data-description="env['ir.model']._get(main_object._name).display_name" t-att-data-id="main_object.id" t-att-data-object="main_object._name" t-att-data-controller="publish_controller">
+            <div t-attf-class="js_publish_management css_publication #{extra_classes} #{main_object.website_published and 'css_published' or 'css_unpublished'}" t-att-data-description="env['ir.model']._get(main_object._name).display_name" t-att-data-id="main_object.id" t-att-data-object="main_object._name" t-att-data-controller="publish_controller">
```

Comment:
The problem seems to be the same in stable versions, should not it be fixed there by targeting the class `js_publish_management` (not pretty but ok in stable I guess, especially if with a "TODO adapt in master" comment).

In master, fixing the bug actually is completely useless since we are getting rid of that code (or at least converting it) so this will conflict with our branch for no good reason (you can check `master-website-in-backend-demo` branch, no more website frontend UI :wink:)